packages/lib/RotatingLogs.test.ts

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+import { readdir } from 'node:fs/promises';
+import RotatingLogs from './RotatingLogs';
+import { createTempDir } from './testing/test-utils';
+
+const createEmptyFileOfSize = (fileName: string, size: number) => {
+	return new Promise((resolve, reject) => {
+		try {
+			const fd = fs.openSync(fileName, 'w');
+			fs.writeSync(fd, 'test', size);
+			resolve(true);
+		} catch (error) {
+			reject(error);
+		}
+	});
+};
+
+describe('RotatingLogs', () => {
+	let dir: string;
+	let files: string[];
+	let rotatingLogs: RotatingLogs;
+
+	beforeAll(async () => {
+		dir = await createTempDir();
+		rotatingLogs = new RotatingLogs(dir);
```

Comment:
Use the constructor variables to set a different max file size. That way you don't need to create huge files for testing